cpp/ScanChain_Zp.C

```diff
@@ -33,9 +34,13 @@
 // #define DEBUG
 
 bool muonDebug = false;
+bool useTuneP = true;
+bool usePuppiMET = true;
```

Comment:
Any particular reason to use the PUPPI MET by default? I have not seen it recommended over PF MET (sorry if I have missed the latest recommendations), so I understand this is a matter of choice? I see that an advantage of using PUPPI MET is the simplification of the systematic uncertainties, since they are precomputed in NanoAOD.

Reply:
Another reason to use puppi MET as default also to avoid issues with tuneP muon pT, in case we decide to use MET.
But we can easily revert to PF MET by switching this line to false.